backend/Dockerfile

```diff
@@ -14,11 +14,21 @@ COPY . .
 RUN apk add --update gcc musl-dev
 RUN go build -o /bin/apiserver backend/src/apiserver/*.go
 
-FROM python:3.5.0-slim as compiler
+FROM python:3.5 as compiler
+
+RUN apt-get update -y && \
+    apt-get install --no-install-recommends -y -q default-jdk libssl-dev libffi-dev wget 
```

Comment:
are libssl-dev and libffi-dev needed?
doesn't seem to be needed for travis but i could be wrong. 
https://github.com/kubeflow/pipelines/blob/master/.travis.yml#L20